docs/microsoft-365-developer-program-faq.yml

```diff
@@ -270,7 +272,9 @@ sections:
       - question: |
           As a Microsoft Partner, can I receive a subscription? 
         answer: |
-          Yes, you can follow [these instructions](microsoft-365-developer-program.md) to join the Microsoft 365 Developer Program and set up a Microsoft 365 developer subscription. However, if you already have a Visual Studio Enterprise or Visual Studio Professional subscription, you can get a Microsoft 365 developer subscription as a benefit. To access this, go to [Visual Studio | My Benefits](https://my.visualstudio.com/benefits). For more information, contact [Visual Studio customer service](https://www.visualstudio.com/subscriptions/support/). 
+          All Developer Program members must qualify for a Microsoft 365 E5 subscription. If you're a qualified member, you'll be invited to set up a developer subscription after you join the program. If you don't immediately qualify, after you join the program, you'll have an option to submit additional information to validate that you're a qualified member. Submit the information and if you qualify, you'll receive an email inviting you to set up your subscription within two weeks. If you don't receive an email, you don't currently qualify for a subscription.
```

Comment:
Do we need to tell them how to submit qualifications?  A link to the form?

Reply:
We can, but how they submit will change as we add the UI experience. The form should be the short-term solution. So, maybe we can be more generic here.